src/constants/endpoints.ts

```diff
@@ -5,6 +5,13 @@ export const API_ENDPOINT_URLS = {
   [BscChainId.TESTNET]: 'https://testnetapi.venus.io/api',
 };
 
+export const SUBGRAPH_ENDPOINT_URLS = {
+  [BscChainId.MAINNET]:
+    'https://api.thegraph.com/subgraphs/name/venusprotocol/venus-isolated-pools',
+  [BscChainId.TESTNET]:
+    'https://api.thegraph.com/subgraphs/name/venusprotocol/venus-isolated-pools-chapel',
+};
+
```

Comment:
These can be removed since we use an environment variable to grab the endpoint URL